pokemon-api/src/main/java/cz/muni/fi/pa165/pokemon/league/participation/manager/dto/ChangeGymTypeDTO.java

```diff
@@ -0,0 +1,65 @@
+package cz.muni.fi.pa165.pokemon.league.participation.manager.dto;
+
+import java.util.Objects;
+import javax.validation.constraints.NotNull;
+
+/**
+ * DTO for changing type of the gym. 
+ * 
+ * @author Tamás Rózsa 445653
+ */
+public class ChangeGymTypeDTO {
+    
+    @NotNull
+    private Long gymId;
+    
+    @NotNull
+    private PokemonType newGymType;
+
```

Comment:
How will you know whether the requesting person is actually the gym leader of the said gym when you don't know who requests the change?

Reply:
That snippet shown is misleading, this comment concerns the whole DTO.